selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml

```diff
@@ -576,12 +576,6 @@
             <class name="org.eclipse.che.selenium.projectexplorer.CreateNewNotJavaFilesTest"/>
             <class name="org.eclipse.che.selenium.projectexplorer.CreateNewPackageFromContextMenuTest"/>
             <class name="org.eclipse.che.selenium.projectexplorer.CreateNewPackagesWithHelpCreationJavaClassTest"/>
-            <class name="org.eclipse.che.selenium.projectexplorer.CreatePackagesAndFilesWithHelpIconCreateTest">
```

Comment:
Please, don't forget to delete those tests CreatePackagesAndFilesWithHelpIconCreateTest, DeleteFilesPackagesAndProjectWithHelpDeleteIconTest  from test suites https://github.com/codenvy/codenvy/blob/master/selenium/codenvy-selenium-test/src/test/resources/suites/CodenvyOnpremSuite.xml#L522l and https://github.com/codenvy/saas/blob/master/selenium/saas-selenium-test/src/test/resources/suites/CodenvySaasSuite.xml#L531 after the merge changes into the master.